plugins/typescript-semantics.js

```diff
@@ -18,11 +18,17 @@ module.exports = {
     'no-return-await': 0,
     'no-throw-literal': 0,
     'require-await': 0,
+    'key-spacing': 0,
 
     // disallows awaiting a value that is not a `Thenable`
     '@typescript-eslint/await-thenable': 2,
     // enforce dot notation whenever possible
     '@typescript-eslint/dot-notation': bestPractices['dot-notation'],
+    // enforce consistent spacing between keys and values in object literal properties
+    '@typescript-eslint/key-spacing': [2, {
+      beforeColon: false,
+      afterColon: true
+    }],
```

Comment:
```suggestion
    '@typescript-eslint/key-spacing': stylisticIssues['key-spacing'],
```

with `stylisticIssues` require'd at top of file.